ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -547,11 +548,7 @@ Item {
                 anchors.topMargin: isEmoji ? 2 : 0
                 anchors.left: parent.left
                 anchors.right: parent.right
-                // using a padding instead of a margin let's us select text more easily
-                anchors.leftMargin: -leftPadding
-                textField.leftPadding: leftPadding
```

Comment:
@jrainville could you please check this whether is ok to remove, it was introduced in https://github.com/status-im/status-desktop/commit/bffe888b0db68a5442e2acd6121008cde636aee5